src/Application/Config/Option.php

```diff
@@ -57,4 +66,21 @@ private function __construct()
     {
         // Don't allow instantiation.
     }
+
+    public static function getYamlOptionMap()
+    {
+        return [
+            'root_dir' => self::ROOT_DIR,
+            self::YAML_DB_HOST => self::DB_HOST,
+            self::YAML_DB_NAME => self::DB_NAME,
+            self::YAML_DB_USER => self::DB_USER,
+            self::YAML_DB_PASS => self::DB_PASS,
+            self::YAML_DB_PORT => self::DB_PORT,
+            'table_groups' => self::TABLE_GROUPS,
+            'tmp_dir' => self::TEMPORARY_DIR,
+            self::YAML_STORAGE_SECRET_KEY => self::STORAGE_SECRET_KEY,
+            'data_bucket' => self::STORAGE_DATA_BUCKET,
+            self::YAML_STORAGE_ACCESS_KEY => self::STORAGE_ACCESS_KEY,
+        ];
```

Comment:
Do we need this map, considering that the transformation is always `_` to `-` for every option?

Reply:
I was thinking about just doing an str_replace at first this seemed a bit more bespoke and obvious. if you think a replace would work better I can refactor it.

Reply:
yup, that's fair. I'll do that today